ops/storage.py

```diff
@@ -20,12 +20,22 @@
 import subprocess
 import typing
 from datetime import timedelta
+from typing import Any, Callable, Generator, List, Optional, Tuple, Type, Union
 
 import yaml
 
+if typing.TYPE_CHECKING:
+    from pathlib import Path
 
-def _run(args, **kw):
-    cmd = shutil.which(args[0])
+    _Notice = Tuple[str, str, str]
```

Comment:
Maybe we should add a comment here for what these `str`s represent.

Reply:
done